specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/account.json

```diff
@@ -0,0 +1,1085 @@
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "description": "The list of user identities associated with the resource. The user identity dictionary key references will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.",
+          "additionalProperties": {
+            "$ref": "#/definitions/UserAssignedIdentitiesProperties"
+          }
+        }
+      }
+    },
+    "EncryptionProperties": {
+      "description": "The encryption settings for automation account",
+      "properties": {
+        "keyVaultProperties": {
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Key vault properties."
+        },
+        "keySource": {
+          "type": "string",
+          "description": "Encryption Key Source",
+          "enum": [
+            "Microsoft.Automation",
+            "Microsoft.Keyvault"
+          ],
+          "x-ms-enum": {
+            "name": "EncryptionKeySourceType",
+            "modelAsString": false
+          }
+        },
+        "identity": {
+          "description": "User identity used for CMK.",
+          "properties": {
+            "userAssignedIdentity": {
+              "type": "object",
+              "description": "The user identity used for CMK. It will be an ARM resource id in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'."
+            }
+          }
+        }
+      }
+    },
+    "KeyVaultProperties": {
+      "type": "object",
+      "description": "Settings concerning key vault encryption for a configuration store.",
+      "properties": {
+        "keyvaultUri": {
+          "description": "The URI of the key vault key used to encrypt data.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of key used to encrypt data.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The key version of the key used to encrypt data.",
+          "type": "string"
+        }
+      }
+    },
+    "DeletedRunbookProperties": {
+      "type": "object",
+      "properties": {
+        "runbookId": {
+          "type": "string",
+          "description": "Gets or sets the Runbook Id."
+        },
+        "runbookType": {
+          "type": "string",
+          "description": "Type of the runbook."
+        },
+        "runtime": {
+          "type": "string",
+          "description": "Gets or sets runtime of the runbook.",
+          "x-nullable": true
```

Comment:
>   "x-nullable": true [](http://example.com/codeflow?start=7&length=21)

As a general rule, we don't recommend for azure APIs, using x-nullable, which results in explicit 'null' in the response. Just omitting the property is preferred.